### PR TITLE
H009_migration_to_java_15_hardcoded_elements_replacement_by_variables

### PR DESCRIPTION
**Pull request by droanna**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,9 +15,9 @@
 	<description>Project hospital - Arcade IT</description>
 
 	<properties>
-		<java.version>11</java.version>
-		<maven.compiler.source>11</maven.compiler.source>
-		<maven.compiler.target>11</maven.compiler.target>
+		<java.version>15</java.version>
+		<maven.compiler.release>${java.version}</maven.compiler.release>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<thymeleaf.version>3.0.11.RELEASE</thymeleaf.version>
```
Comment (staniakm) [inline pom.xml:21]: Unused property

File: pom.xml [hunk 2]
```diff
@@ -76,12 +78,12 @@
 		<dependency>
 			<groupId>org.thymeleaf.extras</groupId>
```
Comment (staniakm) [inline pom.xml:79]: Thymeleaf is not used in project. I think that there will be react as a rest api consumer/ frontend library

Comment (droanna) [reply pom.xml:79]: It is used for Security.

Comment (staniakm) [reply pom.xml:79]: As far as i know it is used only when Thymeleaf template engine is used. Here, we have external, react library for UI layer 
and spring-boot-starter-security is enough.

Comment (rwedzony) [reply pom.xml:79]: Thymeleaf is used for security, but only when you use template with it and you want to use some tags like: <div sec:authorize>. As we don't use thymeleaf at all you can just delete it.

Comment (droanna) [reply pom.xml:79]: Ok, we can discuss security, but for now there is a class in code for which this dependecy is needed and is used (SecurityConfiguration.class). Removing this dependency from pom.xml will break the application build. I think that approach to security should be solved in other task.

Comment (staniakm) [reply pom.xml:79]: Application is broken anyway. There is a list in entity class without any mapping annotation so application is not starting correctly. Imo, just remove import, fix class that is using that import and close PR. We are just wasting time discusing about minor things.

Comment (droanna) [reply pom.xml:79]: "Application is broken anyway", but it's not the explenation for adding new failures. "Fixing" class in this case means removing this class and I will not do this without knowledge of person, who added it. As I wrote before, I think it shouldn't be done here. Security is not minor thing.

**Pull request closed without merging**